spec/models/course/forum/topic_spec.rb

```diff
@@ -10,18 +10,21 @@
   with_tenant(:instance) do
     describe '#slug_candidates' do
       let(:forum) { create(:forum) }
-      let!(:first_topic) { create(:forum_topic, title: 'slug') }
-      let!(:second_topic) { create(:forum_topic, title: 'slug', forum: forum) }
 
       context 'when title is unique' do
         it 'generates a slug based on the title' do
-          expect(first_topic.slug).to eq('slug')
+          unique_name = 'slug' + Time.zone.now.to_formatted_s(:number)
+          topic = create(:forum_topic, title: unique_name)
+          expect(topic.slug).to eq(unique_name)
         end
       end
 
       context 'when title is not unique but forum id is unique' do
         it 'generates a slug based on title and forum_id' do
-          expect(second_topic.slug).to eq("slug-#{forum.id}")
+          unique_name = 'second_slug' + Time.zone.now.to_formatted_s(:number)
```

Comment:
It won't be the same, because the test order is random.

Do you need to test this, actually?